src/stratigraphy/metadata/metadata.py

```diff
@@ -107,6 +107,14 @@ def __init__(self):
         """Initializes the StratigraphyMetadata object."""
         self.metadata_per_file = []
 
+    def add_metadata(self, metadata: BoreholeMetadata):
```

Comment:
The question for the metadata to me is whether we would like to copy the behavior we now have for the layers and groundwatr with having:

Elevation
ElevationOnPage
ElevationInDocument
OverallElevation

Or if we keep everything in the Metadata Object.